src/Transports.AspNetCore/WebSockets/BaseSubscriptionServer.Observer.cs

```diff
@@ -0,0 +1,88 @@
+namespace GraphQL.Server.Transports.AspNetCore.WebSockets;
+
+public abstract partial class BaseSubscriptionServer
+{
+    /// <summary>
+    /// Handles messages from the event source.
+    /// </summary>
+    private class Observer : IObserver<ExecutionResult>
+    {
+        private readonly BaseSubscriptionServer _handler;
```

Comment:
```suggestion
        private readonly BaseSubscriptionServer _server;
```

We already have processor interface. Let's no introduce new terms.